libibnetdisc/src/ibnetdisc.c

```diff
@@ -698,7 +709,12 @@ void add_to_portlid_hash(ibnd_port_t * port, GHashTable *htable)
 		/* We add the port for all lids
 		 * so it is easier to find any "random" lid specified */
 		for (lid = base_lid; lid <= (base_lid + lid_mask); lid++) {
-			g_hash_table_insert(htable, GINT_TO_POINTER(lid), port);
+			struct lid2guid_item *item;
+
+			item = malloc(sizeof(*item));
```

Comment:
calloc?  Or whatever the current rdma-core preferred alloc is...

Reply:
all members should be initialized..